apps/pie-storybook/stories/pie-input.stories.ts

```diff
@@ -218,7 +238,9 @@ const Template = ({
             ?autoFocus="${autoFocus}"
             ?readonly="${readonly}"
             @input="${onInput}"
-            @change="${onChange}">
+            @change="${onChange}"
+            assistiveText="${ifDefined(assistiveText)}"
+            status=${status || nothing}>
```

Comment:
Nit: Can we keep the events grouped at the end?
```suggestion
            assistiveText="${ifDefined(assistiveText)}"
            status=${status || nothing}
            @change="${onChange}">
```